types/types.ts

```diff
@@ -0,0 +1,9 @@
+import {ReactNode} from 'react';
```

Comment:
Move this file to Sidebar folder (and delete types folder).
These types are related to sidebar components only